ssl/ssl_local.h

```diff
@@ -2927,10 +2927,6 @@ const EVP_MD *ssl_evp_md_fetch(OSSL_LIB_CTX *libctx,
 int ssl_evp_md_up_ref(const EVP_MD *md);
 void ssl_evp_md_free(const EVP_MD *md);
 
-int tls_provider_set_tls_params(SSL_CONNECTION *s, EVP_CIPHER_CTX *ctx,
-                                const EVP_CIPHER *ciph,
-                                const EVP_MD *md);
-
```

Comment:
Redundant with `ossl_set_tosl_provider_parameter()`?

Reply:
Yes. The code was moved into the record layer as part of the record layer refactor....but the original got left behind.